stories/info-label/info-label.stories.js

```diff
@@ -80,4 +108,11 @@ export const rtl = () => `
     <span class="fd-info-label fd-info-label--accent-color-2 fd-info-label--icon sap-icon--upload-to-cloud"></span>
 </div>
 `;
-rtl.storyName = 'Info Label RTL';
+rtl.storyName = 'RTL';
+rtl.parameters = {
+    docs: {
+        iframeHeight: 500,
+        storyDescription: `The info label can be displayed from right to left on the screen.
```

Comment:
@mikerodonnell89 @droshev do you think it's a good idea to add a description here or keep it just RTL in the title?

Reply:
For me the description is fine.

Reply:
What do you think would be informative to a developer audience?

Reply:
I say yes to a description but mention that it's typically used for internationalization purposes

Reply:
Done.